packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1636,18 +1635,17 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         trackState: boolean,
         /** Logger to use for correlated summary events */
         summaryLogger: ITelemetryLogger,
-    }): Promise<IChannelSummarizeResult> {
+    }): Promise<ISummaryTreeWithStats> {
         const { runGC, fullTree = false, trackState, summaryLogger } = options;
 
         if (runGC) {
             await this.collectGarbage(summaryLogger);
         }
 
         const summarizeResult = await this.summarizerNode.summarize(fullTree, trackState);
-        assert(summarizeResult.summary.type === SummaryType.Tree,
-            0x12f /* "Container Runtime's summarize should always return a tree" */);
```

Comment:
do we want to preserve the tag here? should it just be moved to assertIsSummaryTree